source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -37,6 +37,7 @@
 import copy
 import logging
 import six
+import shortuuid
```

Comment:
maybe the dependence to `uuid` could be removed. Even from the pip requirements if that's used anywhere else.